vanilla-framework/vanilla-brochure-theme
# Add view viewport attr to jekyll template

## Done

Add view viewport attr to jekyll template

## QA

- Run `gulp jekyll`
- Check that patterns can be viewed in different resolutions without zooming
